documentation/book/con-what-the-cluster-operator-does.adoc

```diff
@@ -15,9 +15,15 @@ On {OpenShiftName} such a cluster can be deployed using the Source2Image feature
 image::cluster_operator.png[Cluster Operator]
 
 When the Cluster Operator is up, it starts to _watch_ for certain {ProductPlatformName} resources containing the desired Kafka or Kafka Connect cluster configuration.
-A `Kafka` resource is used for Kafka cluster configuration, and a `KafkaConnect` resource is used for Kafka Connect cluster configuration.
+By default it watches only in the same namespace or project where it is installed.
+The Cluster Operator can be configured to watch for more {Namespaces}.
+The resources Cluster Operator is watching for are:
```

Comment:
- By default it watches > By default, it watches 

- The resources Cluster Operator is watching for are: > Cluster Operator watches the following resources: